daemon/access.go

```diff
@@ -246,3 +247,34 @@ func (ac interfaceAuthenticatedAccess) CheckAccess(d *Daemon, r *http.Request, u
 
 	return Unauthorized("access denied")
 }
+
+type noticeReadAccess struct{}
```

Comment:
I think this defeats a bit the idea that the access checkers are somewhat declarative,  my preferene would be for this to take a map:

noticeType ->  interface names

that drives the behavior of it

Reply:
updated, thank you!